src/viam/api/config/viam-cpp-sdk-libviamapi.pc.in

```diff
@@ -0,0 +1,13 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+base_includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+deep_includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@/viam/api
```

Comment:
We now need two include directories to paper over the fact that proto generation doesn't generate includes with `viam/api` at the front.